$NetBSD: patch-xen_Makefile,v 1.1 2021/04/18 12:31:26 bouyer Exp $

Use a more strict pattern; on NetBSD it includes some extra compiler
outputs.

--- xen/Makefile.orig	2020-12-10 16:06:07.432987067 +0100
+++ xen/Makefile	2020-12-10 16:06:10.485550293 +0100
@@ -411,7 +411,7 @@
 	  echo "#ifndef __ASM_OFFSETS_H__"; \
 	  echo "#define __ASM_OFFSETS_H__"; \
 	  echo ""; \
-	  sed -rne "/^[^#].*==>/{s:.*==>(.*)<==.*:\1:; s: [\$$#]: :; p;}"; \
+	  sed -rne "/^[^#]ascii.*==>/{s:.*==>(.*)<==.*:\1:; s: [\$$#]: :; p;}"; \
 	  echo ""; \
 	  echo "#endif") <$< >$@
 
